packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/amplify-table-manager-handler.ts

```diff
@@ -739,6 +799,32 @@ export const getSseUpdate = (currentState: TableDescription, endState: CustomDDB
   return undefined;
 };
 
+/**
+ * Compares the currentState with the tags on the resource provider lambda to determine if the tags are updated
+ * @param currentTags current tags on the table
+ * @param newTags new tags on the lambda
+ * @returns Boolean indicating if the tags are updated
+ */
+export const requiresTagsUpdate = (currentTags: DynamoDBTag[], newTags?: DynamoDBTag[]): boolean => {
+  if (!newTags || newTags.length === 0) {
```

Comment:
This means we won't support deleting all tags. Is that expected?

Reply:
I don't see any reason to disallow this case. I'll update this.

Reply:
Removing tags requires a different API call. I have updated the logic to remove tags.